### ISSUE TITLE
Plans for Enhancing to Utilize New Jekyll 2.0.x?

### ISSUE THREAD

**Issue by sc0ttwad3** [sondr3/generator-jekyllized#46]
With the release of **[Jekyll 2.0](http://jekyllrb.com/news/2014/05/06/jekyll-turns-2-0-0/)** a couple of days ago are there any plans, initial ideas, work-in-progress around updating _Jekyllized_ to incorporate and utilize this new major version of Jekyll? 

I'm excited by the quality and progress of this project/generator and would like to participate/contribute to its on-going evolution.  What changes or enhancements do you envision for _Jekyllized_  now that it's core component has been significantly updated, opening up many possibilities.   With this new release, for example, the Jekyllized roadmap _includes possible CoffeeScript support_ and it turns out one major new feature in Jekyll among many others is **[Jekyll 2.0.x native SASS and CoffeeScript support](http://jekyllrb.com/docs/assets/)**.

In short, I can see other areas and features now possible with the functionality added and/or enhanced in Jekyll 2.0.x that I'm wondering about where they might fit into an updated roadmap for Jekyllized that you may have in mind?
